events/memberJoined.js

```diff
@@ -2,9 +2,9 @@ const moment = require('moment');
 const format = require('../momentFormat');
 
 module.exports = {
-  process: (bot, guild, member) => {
-    const userLogsChannel = guild.channels.find('name', 'user-logs');
-    const welcomeChannel = guild.channels.find('name', 'welcome-room');
+  process: (bot, member) => {
```

Comment:
That was part of their changes in 9 -> 10. You can derive `guild` from `member.guild` so they stopped passing it to the event handler. (See under the heading "Events" on the link in the OP.)